src/renderers/webgl/WebGLTextures.js

```diff
@@ -780,7 +780,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 					state.texStorage2D( _gl.TEXTURE_2D, 1, glInternalFormat, image.width, image.height );
 
-				} else {
+				} else if ( allocateMemory ) {
```

Comment:
It's indeed more correct. A `DepthTexture` is considered to be immutable. So after its creation, it can't be changed anymore. Setting `needsUpdate` to `true` makes no sense.

Do you mind changing the code section to:
```js
if ( allocateMemory ) {

    if ( useTexStorage ) {

        state.texStorage2D( _gl.TEXTURE_2D, 1, glInternalFormat, image.width, image.height );

    } else {

        state.texImage2D( _gl.TEXTURE_2D, 0, glInternalFormat, image.width, image.height, 0, glFormat, glType, null );

    }

}
```
Can you also apply this pattern to the `FramebufferTexture` section?